terraform/schemas.go

```diff
@@ -0,0 +1,34 @@
+package terraform
+
+import (
+	"github.com/hashicorp/terraform/config/configschema"
+)
+
+type Schemas struct {
+	Providers ProviderSchemas
```

Comment:
(this seemingly-unnecessary container will grow to include provisioner schemas in a later commit)